core/primitives/Cargo.toml

```diff
@@ -33,6 +33,14 @@ substrate-serializer = { path = "../serializer" }
 pretty_assertions = "0.6"
 hex-literal = "0.2"
 rand = "0.6"
+criterion = "0.2"
+
+[[bench]]
+name = "benches"
+harness = false
+
+[lib]
+bench = false
```

Comment:
[reason](https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options)